woocommerce/woocommerce-android
# [Hazmat Shipping] Introduce the hazmat section description

Summary
==========
Fix issue #9574 by creating the Hazmat description with the expected links and actions. This PR does not yet connect the selected Hazmat category with the Shipping Label creation. It will be introduced at https://github.com/woocommerce/woocommerce-android/pull/9645. This one is just UI work.

Captures
==========
https://github.com/woocommerce/woocommerce-android/assets/5920403/572adff6-16f6-434d-a43b-b9c23c351d4e

How to Test
==========
1. Open the order details of an order eligible for Shipping label creation
2. Start the shipping label creation flow until you reach the Package details section
3. Inside the package details view, make sure the Contains hazardous materials toggle is visible and interactable
4. Make sure the toggle expands and collapses a basic view that will serve as the main control options for the HAZMAT declaration
5. Hit the highlighted links inside the description sections, make sure they're all selectable, and trigger the internal webview.

Update release notes:
